open_discussions/templatetags/noindex_meta.py

```diff
@@ -12,6 +11,7 @@ def noindex_meta():
     """Add noindex for non-production environments."""
     return (
         format_html("""<meta name="robots" content="noindex">""")
-        if settings.ENVIRONMENT not in ("production", "prod")
-        else ""
+        # Restore conditional inclusion after launch
+        # if settings.ENVIRONMENT not in ("production", "prod")
+        # else ""
```

Comment:
I considered adding another environment variable, `DISALLOW_SEARCH_ENGINE_INDEXING` (or something) to control this. But the env check seems good. So then we'd have two checks, one for environment and one for `DISALLOW_SEARCH_ENGINE_INDEXING`. And we'd probably want to get rid of `DISALLOW_SEARCH_ENGINE_INDEXING` after launch, since it would be unused.

So... I just did this, since it seemed simpler.